pkgs/applications/networking/owncloud-client/libre-graph-api-cpp-qt-client.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, mkDerivation
+, cmake
+, qtbase
+}:
+
+mkDerivation rec {
```

Comment:
This should be stdenv.mkDerivation

Reply:
oh, okay, will read upon the differences more.

Reply:
There is not much of difference except I think some cross compilation or internal code but it is considered legacy.